<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAssayShowPostion</class>
 <widget class="BaseDlg" name="QAssayShowPostion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>930</height>
   </rect>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>1071</width>
     <height>141</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabBarAutoHide">
    <bool>true</bool>
   </property>
   <property name="tabs" stdset="0">
    <string notr="true">sub</string>
   </property>
   <widget class="QWidget" name="ch">
    <attribute name="title">
     <string>生化</string>
    </attribute>
    <widget class="QCheckBox" name="ch_add_sample_ckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>761</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>自定义加样顺序</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="checks" stdset="0">
      <string notr="true">push</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="im">
    <attribute name="title">
     <string>免疫</string>
    </attribute>
    <widget class="QCheckBox" name="im_add_sample_ckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>380</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>自定义加样顺序</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="checks" stdset="0">
      <string notr="true">push</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="continuous_sample_ckbox">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>705</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>稀释液ID相同且稀释倍数相同连续加样</string>
     </property>
     <property name="checks" stdset="0">
      <string notr="true">push</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>977</x>
     <y>833</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>851</x>
     <y>833</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QWidget" name="sub_widget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>210</y>
     <width>1058</width>
     <height>585</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>585</height>
    </size>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">all</string>
   </property>
   <widget class="QTableView" name="assay_tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1</y>
      <width>662</width>
      <height>584</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView QScrollBar::handle:vertical  {
    margin: 46px 3px 3px 2px;
}

QTableView QScrollBar::sub-line:vertical  {
    height: 43px; 
    border-bottom: 1px solid #a4a4a4;
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="bordersB" stdset="0">
     <string notr="true">bottom</string>
    </property>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>45</number>
    </attribute>
   </widget>
   <widget class="QFrame" name="line_v_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1</y>
      <width>1</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lines" stdset="0">
     <string notr="true">solid</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>70</y>
      <width>118</width>
      <height>452</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>50</number>
     </property>
     <item>
      <widget class="QPushButton" name="top_btn">
       <property name="minimumSize">
        <size>
         <width>116</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>置顶</string>
       </property>
       <property name="buttons" stdset="0">
        <string notr="true">btn_dlg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="up_btn">
       <property name="minimumSize">
        <size>
         <width>116</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>向上</string>
       </property>
       <property name="buttons" stdset="0">
        <string notr="true">btn_dlg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="down_btn">
       <property name="minimumSize">
        <size>
         <width>116</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>向下</string>
       </property>
       <property name="buttons" stdset="0">
        <string notr="true">btn_dlg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bottom_btn">
       <property name="minimumSize">
        <size>
         <width>116</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>置底</string>
       </property>
       <property name="buttons" stdset="0">
        <string notr="true">btn_dlg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reset_btn">
       <property name="minimumSize">
        <size>
         <width>116</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>重置</string>
       </property>
       <property name="buttons" stdset="0">
        <string notr="true">btn_dlg</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="line_v_5">
    <property name="geometry">
     <rect>
      <x>661</x>
      <y>1</y>
      <width>1</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lines" stdset="0">
     <string notr="true">solid</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>128</y>
     <width>1080</width>
     <height>678</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">all</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>QAssayShowPostion</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>802</x>
     <y>801</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>811</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
